src/commands.cc

```diff
@@ -1251,9 +1251,9 @@ const CommandDesc unset_option_cmd = {
 };
 
 const CommandDesc declare_option_cmd = {
+    "declare",
```

Comment:
I think it's better if we keep this command name a verb, in the name of consistency.